src/assets/styles/theme-tailwind.css

```diff
@@ -1083,12 +1083,10 @@
   border-radius: var(--border-radius);
 }
 .p-fieldset .p-fieldset-legend {
-  padding: 1.25rem;
-  border: 1px solid var(--surface-d);
+  padding: 0.5rem;
   color: var(--text-color);
-  background: var(--surface-b);
   font-weight: 700;
-  border-radius: var(--border-radius);
+  font-size: 20px;
```

Comment:
Can I suggest that this font-size use a relative unit (1.25rem)?

Reply:
yes